requirements-dev.txt

```diff
@@ -3,3 +3,8 @@ pypistats
 pytest>=4.6
 pytest-cov
 responses
+pyright
+pandas-stubs
+types-docutils
+types-requests
+types-tqdm
```

Comment:
Some libraries don't come with type annotations, but awesome contributors have packaged them separately. Sometimes these are not complete. The official place for external type annotations is [typeshed](https://github.com/python/typeshed). Those are the ones prefixed with `types-` in this list.